packages/tools/accessibility/src/HtmlTwin/htmlTwinHostComponent.tsx

```diff
@@ -225,7 +277,11 @@ export class HTMLTwinHostComponent extends React.Component<IHTMLTwinHostComponen
         return result;
     }
 
-    private _isGUI(node: Node) {
+    private _isFullscreenGUI(texture: BaseTexture) {

```

Comment:
You should be able to declare it like this:
```typescript
private _isFullscreenGUI(texture: BaseTexture): texture is AdvancedDynamicTexture {
```
That way, when calling `if (this._isFullscreenGUI(texture)) { ... }` you don't have to cast texture `as AdvancedDynamicTexture` in the `if` block.